test/manifests.go

```diff
@@ -156,6 +161,13 @@ const ManifestJSON string = `{
 					"Port": "40000",
 					"Addr": "example.com"
 				}
+			],
+			"Incoming": [
+				{
+					"Port": "8080",
+					"Cert": "{{ pem .Secrets.cert_shared.Cert }}",
+					"Key": "{{ pem .Secrets.cert_shared.Private }}"
```

Comment:
```suggestion
					"Cert": "cert_shared"
```
We should be able to get the secret by its name.